scripts/booklist.js

```diff
@@ -13,33 +13,27 @@ function getBooked(url){
   // Gets the data for each book on the wikipedia url
   get_ia_books(url).then((data)=>{
     $(".loader").hide();
-    if(data instanceof Array){ //checks if any ISBNs found
-      for(let book of data){  // Iterate over each book to get data
-        let isbn = Object.keys(book)[0];
-        if (book[isbn]) {
-          const OLID = Object.keys(book[isbn][isbn].responses)[0];
-          const archiveIdentifier = book[isbn][isbn].responses[OLID]['identifier'];
-          if (archiveIdentifier) {
-            getMetadataFromArchive(archiveIdentifier);
-          } else {
-            getMetadataFromOpenLibrary(OLID, isbn);
-          }
-        }
-      }
-    } else if (data['status'] === 'error') {
+    if (data['status'] === 'error') {
       let p = document.createElement("p");
       p.appendChild((document.createTextNode(data.message)));
       resultsTray.setAttribute("style", "grid-template-columns:none;");
       resultsTray.appendChild(p);
```

Comment:
Since we already include `scripts/build.js` in `booklist.html` we have jQuery.
You can rewrite this using jQuery in 1 line of code.